### PR TITLE
[Next.js][Multi-project] Allow setting default PROJECT environment variable

### PR DESCRIPTION
**Pull request by illiakovalenko**
- [x] This PR follows the [Contribution Guide](https://github.com/Sitecore/jss/blob/dev/CONTRIBUTING.md)
- [ ] Changelog updated

## Description / Motivation
Provides a way for developers to set fallback _PROJECT_ environment variable in case the project is not set for the site
When the environment variable is not set, in this case, _site.project_ is an empty string (in case site doesn't have specified project)

## Testing Details

- [ ] Unit Test Added
- [x] Manual Test/Other - two cases are tested by logging _site.project_ in a catch-all route
  * left PROJECT env variable empty
  * set PROJECT env variable

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: packages/create-sitecore-jss/src/templates/nextjs/src/lib/site-resolver/plugins/default.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ class DefaultPlugin implements SiteResolverPlugin {
     sites.unshift({
       name: config.jssAppName,
       language: config.defaultLanguage,
+      project: config.project,
```
Comment (art-alexeyenko) [inline packages/create-sitecore-jss/src/templates/nextjs/src/lib/site-resolver/plugins/default.ts:11]: Should we modify the SiteInfo type and request project field via GraphQL too? Or does that change come later?

**Review by art-alexeyenko — COMMENTED** [review]
Looks good, just one question below.

File: packages/create-sitecore-jss/src/templates/nextjs/src/lib/site-resolver/plugins/default.ts (hunk 1 above)
Comment (illiakovalenko) [reply packages/create-sitecore-jss/src/templates/nextjs/src/lib/site-resolver/plugins/default.ts:11]: Valid point! It was missed, but was done in POC

**Review by art-alexeyenko — APPROVED** [review]
👍

**Pull request merged**